clients/admin-ui/src/features/integrations/configure-monitor/ConfigureMonitorDatabasesForm.tsx

```diff
@@ -58,6 +76,10 @@ const ConfigureMonitorDatabasesForm = ({
 
   const saveIsDisabled = !allSelected && selected.length === 0;
 
+  if (initialIsLoading) {
+    return <FidesSpinner my={12} />;
```

Comment:
is this meant to center it? If so, wouldn't using auto margins be a safer bet there?
```suggestion
    return <FidesSpinner my="auto" />;
```

Reply:
This is just to give it some room to breathe when it's the only thing in the modal.

No margin or auto:
![Screenshot 2024-11-20 at 11 29 15](https://github.com/user-attachments/assets/abdc3fa1-76e1-4363-8d24-efd15f83c6db)

Constant margin:
![Screenshot 2024-11-20 at 11 29 30](https://github.com/user-attachments/assets/2d759403-f246-4aee-9bce-2c250e3b3e6f)

Reply:
Oh man. My bad. I confused my `x` and `y` lol.